### ISSUE TITLE
SQLTrackerStore url configuration is a misnomer

### ISSUE THREAD

**Issue by jgarber** [RasaHQ/rasa#3614]
**Content of endpoints file (endpoints.yml)**:
```yml
tracker_store:
  type: sql
  dialect: "postgresql"
  url: ${DATABASE_HOST}
  username: ${DATABASE_USER}
  db: rasa
```

The naming of `url` is unfortunate when it's really the hostname. `url` ought to accept a URL like `postgres://postgres:5432`

For backward compatibility, you could look for `://` in the URL and pass it straight through to `sqlalchemy.engine.URL.make_url` in [tracker_store.py](https://github.com/RasaHQ/rasa/blob/6aab6db02188e63dcbd7568a3eb11f951faa7414/rasa/core/tracker_store.py#L42)

**Comment by akelad** [comment]
Thanks for raising this issue, @wochinge  will get back to you about it soon.

**Comment by wochinge** [comment]
@jgarber Good point!

**Comment by wochinge** [comment]
closed by #3715
